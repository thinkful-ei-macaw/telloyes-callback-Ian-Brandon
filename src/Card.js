import React from 'react';
import './Card.css';

export default function Card(props) {
  return (
    <div className='Card'>
      <button
        type='button'
        onClick={() => props.handleDelete(props.id)}
      >
        delete
      </button>
    </div>
  )
}
