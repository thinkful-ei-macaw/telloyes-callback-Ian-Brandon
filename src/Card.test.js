import React from 'react';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import Card from './Card';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Card
    title='test'
    content='also test' />, div);
  ReactDOM.unmountComponentAtNode(div);
})

it('renders Cards as expected', () => {
  const tree = renderer
    .create(<Card
      key='5'
      title='title'
      content='content'
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});