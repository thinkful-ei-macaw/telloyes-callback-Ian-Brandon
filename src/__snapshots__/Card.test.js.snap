// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Cards as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
</div>
`;
